docs/services/base-rc/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Base Remote Control Service"
+linkTitle: "Base Remote Control"
+weight: 80
```

Comment:
Let's put this in alphabetical order (aside from robot - I think that one can stay at the top since it makes sense as the first one)

Reply:
Done!